### PR TITLE
Connections

### PR DESCRIPTION
**Pull request by edlouth**

### CODE REVIEW

File: grai-server/app/connections/apps.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class ConnectionConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
```
Comment (ieaves) [inline grai-server/app/connections/apps.py:5]: This doesn't matter at all but we can probably get by with a simple AutoField for these.

File: grai-server/app/connections/models.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+class Connection(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    connector = models.ForeignKey(
+        "Connector", related_name="connections", on_delete=models.PROTECT
+    )
+    namespace = models.CharField(max_length=255, default="default")
+    name = models.CharField(max_length=255)
+    metadata = models.JSONField(default=dict)
+    secrets = models.JSONField(default=dict, blank=True, null=True)
+    is_active = models.BooleanField(default=True)
```
Comment (ieaves) [inline grai-server/app/connections/models.py:33]: What does `is_active` mean for the connectors? Is it forward looking for some sort of scheduled run capacity?

Comment (edlouth) [reply grai-server/app/connections/models.py:33]: Indeed that was my thought, not that I put a schedule field in

File: grai-server/app/connections/migrations/0003_connection_secrets.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1.3 on 2022-12-12 13:13
```
Comment (ieaves) [inline grai-server/app/connections/migrations/0003_connection_secrets.py:1]: squash migraitons

**Comment by ieaves** [comment]
Need to squash a few migrations but otherwise LGTM!!

File: grai-server/app/connections/migrations/0003_connection_secrets.py (hunk 3 above)
Comment (edlouth) [reply grai-server/app/connections/migrations/0003_connection_secrets.py]: Done

**Comment by edlouth** [comment]
Closing in favour of https://github.com/grai-io/grai-core/pull/63

**Pull request closed without merging**